lib/danger/ci_source/circle.rb

```diff
@@ -12,6 +12,10 @@ def self.validates?(env)
         return !env["CIRCLE_PROJECT_USERNAME"].nil? && !env["CIRCLE_PROJECT_REPONAME"].nil?
       end
 
+      def supported_request_sources
```

Comment:
Yup, but the idea was to not have a "safe" default for GitHub, every CI source should explicitly declare which Request Sources it supports.

Fixed!